README.md

```diff
@@ -78,6 +81,96 @@ You must pass call events handler, registeration event handler (optional) and se
    7. **ToggleHold**: Toggle state hold/un-hold state of a call. This state needs to be maintained by the client
    8. **ToggleMute** Toggle state mute/un-mute state of a call. This state needs to be maintained by the client
 
+
+
+
+
+# For Non-npm integration
+If you are not using a package manager like NPM, you can directly include the bundled JavaScript file that can be generated by webpack.
+
+## Steps
+
+1. Locate the webpack config file in the project directory.
```

Comment:
@Anant-exotel 

Add a step to clone the repo.
Then add a step to go to the directory where webpack config is located.
Command to install dependencies that are required to build.
Alternatives to npx (npm or some other package manager)